QuizzApp/Core/AppUtility/AppUtility.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AppUtility.swift
+//  QuizzApp
+//
+//  Created by bakhva  on 19.06.23.
```

Comment:
ეს ჯობია ერთხელ ჩაასწორო და ყველა ფაილში ავტომატურად დაგენერირებულ ტექსტში სწორად გექნება.
```suggestion
//  Created by Bakhva Jakeli on 19.06.23.
```